build_projects/shared-build-targets-utils/Utils/Dirs.cs

```diff
@@ -27,6 +27,7 @@ public static class Dirs
         public static readonly string CorehostLocked = Path.Combine(Output, "corehost", "locked");
         public static readonly string CorehostLocalPackages = Path.Combine(Output, "corehost");
         public static readonly string CorehostDummyPackages = Path.Combine(Output, "corehostdummypackages");
+        public static readonly string SharedFrameworkPublish = Path.Combine(Output, "sharedFrameworkPublish");
```

Comment:
What do you feel about this being under "intermediate", since it really isn't an "output" of the CLI build.  It is just something on the way of getting to the real output.